docs/configuration/backends/docker.md

```diff
@@ -184,6 +185,7 @@ Services labels can be used for overriding default behaviour
 | `traefik.<service-name>.frontend.passHostHeader`  | Overrides `traefik.frontend.passHostHeader`.                                                     |
 | `traefik.<service-name>.frontend.priority`        | Overrides `traefik.frontend.priority`.                                                           |
 | `traefik.<service-name>.frontend.rule`            | Overrides `traefik.frontend.rule`.                                                               |
+| `traefik.<service-name>.frontend.redirect.https`  | Overrides `traefik.frontend.redirect.https`.                                                     |
```

Comment:
The service documentation may be updated too :wink: 

s/`traefik.<service-name>.frontend.redirect.https`  | Overrides `traefik.frontend.redirect.https`./`traefik.<service-name>.frontend.redirect`  | Overrides `traefik.frontend.redirect`.